packaging/app.organicmaps.desktop.metainfo.xml

```diff
@@ -104,6 +104,23 @@
   </screenshots>
 
   <releases>
+   <release version="2024.01.09-5" date="2024-01-09">
+     <description>
+       <p>Highlights:</p>
+         <ul>
+           <li>New OSM maps data as of January 5, 2024</li>
```

Comment:
Styles:
- Added cairns, updated Singapore subway icon
- Slightly increased contrast of lines on the World overview map
- Make secondary roads a bit brighter than tertiary/etc.
- Car navigation style: fixed some footways/paths/tracks being displayed too early as white dashed lines
- Main style: fix color of footways with a sac_scale=* tag to be same as path
- Fixed very thin lines (e.g. tram lines) having inconsistent transparency/contrast between different devices (driver-dependent)
- Fix bookmarked children playground icon
- remove labels and decrease visibility of footway crossings and sidewalks
- tuned contrast of altitude contour lines
- crisper looking lines (no edge blur)

Outdoors style:
- more contrast-looking paths, tracks, cutlines, rivers, ditches, railways, cycleways, fences, etc.
- reduce visibility of small streams and canals
- increase visibility of fences, walls and ditches
- increase visibilities of bridle/cycle/foot-ways and steps, increase label sizes
- make bridleways' dashes longer

Pistes improvements:
- add snow parks and new piste types: skitour, winter hike, connection
- color freeride pistes in yellow-orange, consider ungroomed downhill runs as freeride
- better piste names and numbers/refs support
- in Outdoors style: increase pistes visibilities and display detailed piste shapes (area fills)

Other:
- Display capacity of parkings, bicycle docks and charging stations when they're selected
- Adjust shop, amenity, leisure ranking in search

Linux Desktop:
- set screen update frequency to 60fps

Translations:
- new Azebaijani language
- updates to Chinese, Polish, Hebrew, Romanian

Reply:
Done